python3/raygun4py/utilities.py

```diff
@@ -30,6 +30,11 @@ def filter_keys(filtered_keys, object):
                     if sanitised_key in key:
                         iteration_target[key] = "<filtered>"
 
+    if isinstance(object, raygunmsgs.RaygunMessage):
+        updated_object = object.copy()
+        updated_object.__dict__.update(iteration_target)
+        return updated_object
+
```

Comment:
If the original object was a `RaygunMessage`, then we need to create a copy of the object and apply the changes to the internal `__dict__`